open-source-license-acknowledgements-and-third-party-copyrights.md

```diff
@@ -92,9 +92,9 @@ License: Creative Commons Attribution 3.0 BY
 https://github.com/erikdubbelboer/phpRedisAdmin/blob/master/README.markdown
 
 ### Selenium Docker
-Image: `Selenium/standalone-chrome:3.11`  
-License: Apache License 2.0  
-https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md  
```

Comment:
two spaces are mandatory to preserve formatting 

now it looks like this
![Snímek obrazovky 2023-05-05 v 19 15 43](https://user-images.githubusercontent.com/1177414/236523620-171c3381-be08-4291-96f0-a526717b1829.png)

Reply:
fixed